spec/lib/classroom_management_spec.rb

```diff
@@ -73,5 +79,13 @@
       school = School.find_by(creator_id:)
       expect(Role.teacher.where(user_id: teacher_id, school_id: school.id)).to exist
     end
+
+    # rubocop:disable RSpec/MultipleExpectations
+    it 'assigns students' do
+      school = School.find_by(creator_id:)
+      expect(Role.student.where(user_id: student_1, school_id: school.id)).to exist
+      expect(Role.student.where(user_id: student_2, school_id: school.id)).to exist
+    end
+    # rubocop:enable RSpec/MultipleExpectations
   end
 end
```

Comment:
Is it worth having a test that the students get assigned to the class?

Reply:
great spot, thanks!